dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/metrics/TaskMetrics.java

```diff
@@ -24,7 +24,7 @@
 
 import lombok.experimental.UtilityClass;
 
-import com.facebook.presto.jdbc.internal.guava.collect.ImmutableSet;
+import com.google.common.collect.ImmutableSet;
```

Comment:
remove dependence module `dolphinscheduler-task-all` and some of lib not exists anymore. so we have to change some code